example-deck.gl.md

```diff
@@ -0,0 +1,400 @@
+Plotting CO-Values using deck.gl
+================================
+
+In this example we explain how to use [deck.gl](https://deck.gl/#/),
+a WebGL-powered framework for data visualization created by [Uber](https://www.uber.com),
+to visualize the amount of carbon monoxide in Germany from a couple of days.
+
+The result should look somewhat like this:
+
+<style type="text/css">
+      #container {width: 100%; height: 500px;}
+    </style>
+<div id="container"></div>
```

Comment:
If you use inline-styles anyway, why not simply:

```html
<div id="container" style="height: 500px;"></div>
```

…omitting the `width: 100%;` since that should be the default for block display elements.

Reply:
Fixed that. I thought deck.gl would override the `div` and with it the inline style.